scripts/SDM/selectBackground.yml

```diff
@@ -29,7 +31,20 @@ inputs:
     label: predictors
     description: raster, predictors variables 
     type: image/tiff;application=geotiff[]
-    example: ["/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio141981-01-01.tif", "/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio151981-01-01.tif", "/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio181981-01-01.tif","/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio21981-01-01.tif","/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio31981-01-01.tif","/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio81981-01-01.tif","/output/SDM/loadPredictors_R/e09acd85debd23c991652771b1d771b2/bio91981-01-01.tif"]
+    example: "/scripts/filtering/cleanCoordinates_predictors.tif"
+  raster:
+    label: raster file or taxa
+    description: taxa-specific raster of sampling, used in weighted_raster and unweighted_raster methods, if using from_cube then the corresponding taxa (e.g., 'reptiles')
+    type: text;image/tiff;application=geotiff[]
```

Comment:
Is this really intended to be an array of tiffs?
If yes remove "text;", if not remove both "text;" and "[]"